crates/rustls/src/lib.rs

```diff
@@ -28,17 +28,8 @@
 )]
 #![allow(clippy::single_component_path_imports)]
 
-//! TLS protocol related components for DNS over TLS
+//! Deprecated and removed as of 0.21.0, see the dns-over-rustls feature in `trust-dns-proto`
 
-#[macro_use]
-extern crate log;
-
-pub mod tls_client_stream;
-pub mod tls_server;
-pub mod tls_stream;
-
-pub use self::tls_client_stream::{tls_client_connect, TlsClientStream};
-pub use self::tls_stream::{tls_connect, tls_from_stream, TlsStream};
-
-#[cfg(test)]
-mod tests;
+compile_error!(
```

Comment:
@djc I'm not sure about this one... other option is to deprecate and reexport the original symbols from the proto crate. I highly doubt people are using this crate directly though... so this seems correct.

Reply:
Yeah, this seems okay.

Reply:
I decided to deprecate and reexport to help folks get the message where they might be using these crates directly.